From 7f8c105accbe649bb4300faaf7e9d2707a415313 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Thu, 5 Mar 2026 11:11:10 +0800
Subject: [PATCH 43/49] loonggpu: include linux/version.h in loonggpu_helper.h

Some helpers in this header uses LINUX_VERSION_CODE in #if directives,
include linux/version.h here instead of relying on the translation
units to include them.

Signed-off-by: Xi Ruoyao <xry111@xry111.site>
---
 include/loonggpu_helper.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/loonggpu_helper.h b/include/loonggpu_helper.h
index 7a4690c..144ddee 100644
--- a/include/loonggpu_helper.h
+++ b/include/loonggpu_helper.h
@@ -40,6 +40,7 @@
 #include <drm/drm_irq.h>
 #endif
 #include <drm/drm_modeset_helper.h>
+#include <linux/version.h>
 
 /*
  * drm_sched_hw_job_reset() was replaced with drm_seched_stop.
-- 
2.56.0

